src/hub.js

```diff
@@ -495,6 +503,12 @@ export async function updateEnvironmentForHub(hub, entryManager) {
 
             const fader = document.getElementById("viewing-camera").components["fader"];
 
+            if (isLockedDownDemoRoom()) {
```

Comment:
Trying to find a better place for this, the best place I can think of is here after the `scene_media_selected` listener:
https://github.com/mozilla/hubs/blob/a87f4b937fb4fa3b8badf4f116eb8489b329d72d/src/hub.js#L852

We can add something there like:
```javascript
  scene.addEventListener("hub_updated", async () => {
    if (isLockedDownDemoRoom()) {
      const avatarRig = document.querySelector("#avatar-rig");
      const avatarId = await fetchRandomDefaultAvatarId();
      avatarRig.setAttribute("player-info", { avatarSrc: await getAvatarSrc(avatarId) });
    } else {
      if (scene.is("entered")) {
        entryManager._setPlayerInfoFromProfile(true);
      }
    }
  });
```

This will minimize the number of changes in the codebase a little bit.

Reply:
Amazing! Agreed!